packages/fiori-freestyle-writer/test/__snapshots__/basic.test.ts.snap

```diff
@@ -1,5 +1,668 @@
+  
+      return BaseController.extend(\\"nods1.controller.App\\", {
+        onInit: function() {
+        }
+      });
+    }
+  );
+  ",
+    "state": "modified",
+  },
+  "webapp/controller/View1.controller.js": Object {
+    "contents": "sap.ui.define([
+    \\"sap/ui/core/mvc/Controller\\"
+],
+function (Controller) {
+    \\"use strict\\";
+
+    return Controller.extend(\\"nods1.controller.View1\\", {
+        onInit: function () {
+
+        }
+    });
+});
+",
+    "state": "modified",
+  },
+  "webapp/css/style.css": Object {
+    "contents": "/* Enter your custom styles here */",
+    "state": "modified",
+  },
+  "webapp/i18n/i18n.properties": Object {
+    "contents": "# This is the resource bundle for nods1
+
+#Texts for manifest.json
+
+#XTIT: Application name
+appTitle=App Title
+
+#YDES: Application description
+appDescription=A Fiori application.
+#XTIT: Main view title
+title=App Title",
+    "state": "modified",
+  },
+  "webapp/index.html": Object {
+    "contents": "<!DOCTYPE html>
+<html lang=\\"en\\">
+<head>
+    <meta charset=\\"UTF-8\\">
+    <meta name=\\"viewport\\" content=\\"width=device-width, initial-scale=1.0\\">
+    <meta http-equiv=\\"X-UA-Compatible\\" content=\\"IE=edge\\">
+    <title>App Title</title>
+    <style>
+        html, body, body > div, #container, #container-uiarea {
+            height: 100%;
+        }
+    </style>
+    <script
+        id=\\"sap-ui-bootstrap\\"
+        src=\\"resources/sap-ui-core.js\\"
+        data-sap-ui-theme=\\"sap_belize\\"
+        data-sap-ui-resourceroots='{
+            \\"nods1\\": \\"./\\"
+        }'
+        data-sap-ui-oninit=\\"module:sap/ui/core/ComponentSupport\\"
+        data-sap-ui-compatVersion=\\"edge\\"
+        data-sap-ui-async=\\"true\\"
+        data-sap-ui-frameOptions=\\"trusted\\"
+    ></script>
+</head>
+<body class=\\"sapUiBody sapUiSizeCompact\\" id=\\"content\\">
+    <div
+        data-sap-ui-component
+        data-name=\\"nods1\\"
+        data-id=\\"container\\"
+        data-settings='{\\"id\\" : \\"nods1\\"}'
+        data-handle-validation=\\"true\\"
+    ></div>
+</body>
+</html>",
+    "state": "modified",
+  },
+  "webapp/manifest.json": Object {
+    "contents": "{
+  \\"_version\\": \\"1.22.0\\",
+  \\"sap.app\\": {
+    \\"id\\": \\"nods1\\",
+    \\"type\\": \\"application\\",
+    \\"i18n\\": \\"i18n/i18n.properties\\",
+    \\"applicationVersion\\": {
+      \\"version\\": \\"0.0.1\\"
+    },
+    \\"title\\": \\"{{appTitle}}\\",
+    \\"description\\": \\"{{appDescription}}\\",
+    \\"resources\\": \\"resources.json\\",
+    \\"sourceTemplate\\": {
+      \\"id\\": \\"test-template\\",
+      \\"version\\": \\"1.2.3-test\\"
+    },
+    \\"dataSources\\": {
+      \\"mainService\\": {
+        \\"uri\\": \\"/sap/opu/odata/\\",
+        \\"type\\": \\"OData\\",
+        \\"settings\\": {
+          \\"annotations\\": [],
+          \\"localUri\\": \\"localService/metadata.xml\\",
+          \\"odataVersion\\": \\"2.0\\"
```

Comment:
```suggestion
          \\"odataVersion\\": \\"4.0\\"
```
CAP is V4 by default. 
`annotations` and `localUri` are  not really applicable for CAP as they relate to EDMX annotation files.